### DISCUSSION TITLE
[useFieldArray] - Unregister / Register

### DISCUSSION THREAD

**Discussion by juliopavila** [react-hook-form/react-hook-form#3932] [Q&A]
I need to use the functionality `register` and `unregister` inside the field array the stuck that I have is when I register a field apply inside all fields and no just the row that I've been working

**Comment by bluebill1049** [comment]
are you using shouldUnregister: false or true? by default, when inputs gets unmounted they should be `unregistered`, however, with shouldUnregister: false that's not possible.
